static/specification.html

```diff
@@ -138,605 +336,555 @@ <h2>
+              sessions=], set |success| to false.
+            </p>
+          </li>
+          <li>Otherwise, establish a new [=payment session=].
+          </li>
+          <li>[=Queue an element task=] on the [=networking task source=] given
+          |element| and the following steps:
+            <ol>
+              <li>If |success| is true, [=fire an event=] named `"load"` at
+              |element|.
+              </li>
+              <li>Otherwise, [=fire an event=] named `"error"` at |element|.
+              </li>
+            </ol>
+          </li>
+        </ol>
+      </div>
+      <p>
+        If a "monetization" <code>link</code> <span>becomes browsing-context
+        disconnected</span>, a user agent MUST stop the [=payment session=].
+      </p>
     </section>
-    <section id="flow">
+    <section>
       <h2>
-        Flow
+        `onmonetization` event handler
       </h2>
       <p>
-        This flow refers to the <a>user agent</a> and the user's
-        <a>provider</a>.
+        The `onmonetization` event handler is an [=event handler content
+        attribute=] that can be applied to any [=element=]. The user agent uses
+        it to notify that some [^link^] has been [=monetized=].
       </p>
-      <ol>
-        <li>The <a>user agent</a> sets {{ Monetization / state }} to "{{
-        MonetizationState / pending }}".
+    </section>
+    <section id="events">
+      <h2>
+        monetization event
+      </h2>
+      <p>
+        When the [=payment session=] has sent an |outgoing payment| with a
+        non-zero amount, perform the following steps:
+      </p>
+      <ol class="algorithm">
+        <li>Let |target:HTMLLinkElement| be the {{HTMLLinkElement}} associated
+        with the [=payment session=].
         </li>
-        <li>
-          <p>
-            The <a>user agent</a> observes the [^head^] and looks for a Web
-            Monetization <code>&lt;meta name="monetization"&gt;</code> tag
-            (<a href="#meta-tags-set">specified below</a>).
-          </p>
-          <ul>
-            <li>There MUST be only one <code>&lt;meta
-            name="monetization"&gt;</code> tag at any given time
-            </li>
-            <li>The `&lt;meta&gt;` Tags Set MUST be in the [^head^] of the
-            document.
-            </li>
-            <li>The `&lt;meta&gt;` Tags Set MUST be in the top level window
-            (i.e. not inside an iframe)
-            </li>
-          </ul>
+        <li>If |target| is `null`, then return.
         </li>
-        <li>
-          <p>
-            Below is repeated for every semantically (consider
-            <code>meta.content = meta.content</code>) new <code>&lt;meta
-            name="monetization"&gt;</code> tag seen for the life of the page:
+        <li>Let |eventInitDict:MonetizationEventInit| be a
+        {{MonetizationEventInit}} dictionary, whose members are initialized to
```

Comment:
Elsewhere it says:
```
 The {{MonetizationEvent}} DOM events provide information such as the
          amount sent, the currency of the payment (what we call the "asset
          code"), and a link to the incoming payment at the [=monetization
          receiver=]
```          

The outgoing_payment references an incoming_payment to be attached to the event ?